cyences_app_for_splunk/default/data/ui/views/cs_asset_intelligence.xml

```diff
@@ -390,10 +390,11 @@
       <table>
         <title>Endpoint Service Not Running</title>
         <search>
-          <query>`cs_sophos` type IN ("Event::Endpoint::ServiceNotRunning", "Event::Endpoint::ServiceRestored") | rename dhost as host | eval ip=src_ip | search $tkn_filter_main$ | stats count, latest(_time) as _time, values(src_ip) as src_ip, values(suser) as user, latest(type) as type by host
+          <query>index=main sourcetype="sophos*"  type IN ("Event::Endpoint::ServiceNotRunning", "Event::Endpoint::ServiceRestored") host="*" location IN ("*") source IN ("*")
+| stats count, latest(_time) as _time, values(source_info.ip) as source_info.ip, values(source) as source latest(type) as type by host, location
```

Comment:
rename source_info.ip as src_ip

values(source) as source => is this splunk's metadata field or event's field? if it's extracted from event then it also include the values of splunk's metadata field **source**